domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ClaimDocumentStore.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.cmc.domain.models;
+
+import lombok.Getter;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Getter
+public class ClaimDocumentStore {
```

Comment:
Prefer to rename this, `...Store` doesn't look great in this name.

Reply:
Basically its a collection of ClaimDocument . I didn't want to use Collection because that may be confused with java collection. As an alternative I used , ClaimDocumentStore

Reply:
agree with sabah here - i like collection? :)